roles/virt-install/tests/inventory/host_vars/vm-1.example.com

```diff
@@ -1,15 +1,14 @@
 ---
-
 libvirt_name: "vm-1"
 libvirt_title: "VM 1"
-libvirt_description: "VM used for: XYZ"
+libvirt_description: "VM used for: Testing"
 libvirt_memory: "2048"
-libvirt_vcpus: "4"
-libvirt_disk_size: "5"
+libvirt_vcpus: "1"
+libvirt_disk_size: "20"
 libvirt_os_variant: "f25"
 libvirt_iso: "/tmp/Fedora-Server-dvd-x86_64-25-1.3.iso"
```

Comment:
while you are at it here, please update this to be a "newer" version of Fedora - i.e.: Fedora 28